README.md

```diff
@@ -1,24 +1,14 @@
 # README
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# July 9, 2018
+--- general 
+- updated rails to 5.2.0
 
-Things you may want to cover:
+--- code
+- scaffolded rails app
+- cleaned config/application.rb
+- cleaned gen file
 
-* Ruby version
+--- git
+- merged 1st pull request
 
-* System dependencies
-
-* Configuration
-
-* Database creation
-
-* Database initialization
-
-* How to run the test suite
-
-* Services (job queues, cache servers, search engines, etc.)
-
-* Deployment instructions
-
-* ...
```

Comment:
So these points cover pretty much the things that you need to update the README with, rather what you have done during the day

Reply:
Will this be addressed?

Reply:
I tried to with my readme PR, but I seem to get all the changes in the other files with, even though I pull from upstream development and only edit the README

Reply:
Made new PR with changes to README
https://github.com/CraftAcademy/cibus/pull/11